components/net_traits/lib.rs

```diff
@@ -224,6 +227,11 @@ impl FetchTaskTarget for IpcSender<FetchResponseMsg> {
         let _ = self.send(FetchResponseMsg::ProcessRequestEOF);
     }
 
+    fn process_request_referrer(&mut self, request: &Request) {
+        let _ = self.send(FetchResponseMsg::ProcessRequestReferrer(request.clone().referrer,
+                                                                   request.clone().referrer_policy));
```

Comment:
Cloning the whole request is overkill. We should be able to just clone the referrer and policy values instead.